meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -217,6 +217,7 @@ FEATURE_PACKAGES_tools-benchmark ?= "packagegroup-tools-benchmark"
 FEATURE_PACKAGES_multimedia      ?= "gstreamer1.0 gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad"
 FEATURE_PACKAGES_virtualization  ?= "docker-moby python3-docker-compose"
 FEATURE_PACKAGES_codebench-debug ?= "gdbserver strace openssh-sftp-server"
+FEATURE_PACKAGES_tools-profile:append:feature-tracing ?= " perfetto"
```

Comment:
Drop the soft assignment and instead just the value, as you're using :append here.

Reply:
@Noor-Ahsan Should we move this line to distro layer from BSP layer?
```
FEATUREOVERRIDES .= "${@bb.utils.contains('DISTRO_FEATURES', 'tracing', ':feature-tracing', '', d)}"
```
As now we are supporting tracing at DISTRO level instead of a BSP feature.